common.yml

```diff
@@ -38,12 +33,16 @@
     - name: restart gandalf
       service: name=gandalf-server state=restarted
 
+# Configure all docker nodes in parallel
```

Comment:
We should omit this comment because it's describing Ansible's default behaviour.